### DISCUSSION TITLE
How to tell argocd to orphan a resource?

### DISCUSSION THREAD

**Discussion by hknlof** [argoproj/argo-cd#9954] [Q&A] [answered]
I have resource, that I used to manage with argocd and synced via

```
argocd app sync
```

One of my resources now has a different lifecycle, hence I would like to remove it from my helm chart.

The sync operation now fails with ` OutOfSync  Healthy         ignored (requires pruning)`.

I would like argo not to consider this resource any longer but keep the resource itself in my Kubernetes cluster. How can I achieve this?

**Comment by Roberdvs** [comment, marked as answer]
Couldn't find any info regarding this but what worked for me was removing the label `argocd.argoproj.io/instance` from the resource (in my case a secret) once it was in the `requires pruning` state and then ArgoCD stopped trying to sync it and no longer appeared in the UI.

**Reply by hknlof** [reply]
Thanks, that worked for me as well. Although with StatefulSets, you need to propagate it down resources chain.
